### ISSUE TITLE
Narrator cannot announce the content in RichTextBox

### ISSUE THREAD

**Issue by dmitrii-drobotov** [dotnet/winforms#7387]
### .NET version

7.0 Preview 6

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

This is a regression issue (see https://github.com/dotnet/winforms/issues/3455). It does not repro in .NET 6.0.

### Issue description

Narrator should announce the content in RichTextBox.
**Expected behavior (.NET 6):**
![Screenshot 2022-07-08 123424](https://user-images.githubusercontent.com/102954094/177977789-544c2f8b-3b43-4b57-9cda-545a075d81f9.png)

**Actual behavior (.NET 7 Preview):**
![Screenshot 2022-07-08 123833](https://user-images.githubusercontent.com/102954094/177977827-828322bd-dd0c-4051-bd4c-8da166cff98f.png)

### Steps to reproduce

1. Add a RichTextBox control on a form
2. Enter any text content into RichTextBox, then start Narrator and focus on RichTextBox and observe Narrator behavior

**Comment by John-Qiao** [comment]
Verified with 7.0.100-rc.1.22371.5 from main branch, this issue has been fixed that Narrator announces the content in RichTextBox now.
![TestResult1](https://user-images.githubusercontent.com/45864985/180354587-a0e4fb9b-954f-4ff4-9a05-60afe582e6b8.png)
